src/main/java/gtPlusPlus/xmod/gregtech/loaders/RecipeGenExtruder.java

```diff
@@ -128,22 +122,20 @@ private void generateRecipes(final Material material) {
         }
 
         // Bolt Recipe
-        if (ItemUtils.checkForInvalidItems(material.getIngot(1)) && ItemUtils.checkForInvalidItems(material.getBolt(1)))
-            if (!material.isRadioactive) {
-                GTValues.RA.stdBuilder()
-                    .itemInputs(itemIngot, shape_Bolt)
-                    .itemOutputs(material.getBolt(8))
-                    .duration((int) Math.max(material.getMass() * 2L * 1, 1))
-                    .eut(material.vVoltageMultiplier)
-                    .addTo(extruderRecipes);
+        if (null != (material.getIngot(1)) && null != (material.getBolt(1))) if (!material.isRadioactive) {
```

Comment:
```suggestion
        if (material.getIngot(1) != null && material.getBolt(1) != null && !material.isRadioactive) {
```
Note that this one is different from the others as it combines the two inlined if-statements